### ISSUE TITLE
 GetPlaylistData doesn't return all videos in a playlist

### ISSUE THREAD

**Issue by RodrigoHernandez26** [damonwonghv/youtube-search-api#7]
This playlist has 155 songs, but GetPlaylistData only returns 100

```js
let searchResult = await youtubesearchapi.GetListByKeyword('https://www.youtube.com/playlist?list=PLH4vUK9DCQdnBUbzQTXplfbeLqEpjr7dJ', true)
let playlist = await youtubesearchapi.GetPlaylistData(searchResult.items[0].id, true)
console.log(playlist.items.length) // 100
```

Any playlist with more than 100 songs, GetPlaylistData only returns 100

**Comment by damonwonghv** [comment]
will resolve in version 1.1.1

**Comment by damonwonghv** [comment]
This issue will fix it on 1.1.2. Please wait. Version 1.1.2 will release maybe next week.
